app/src/components/v-field-list/v-field-list.vue

```diff
@@ -36,19 +36,21 @@ interface Props {
 	field?: string;
 	disabledFields?: string[];
 	includeFunctions?: boolean;
+	excludeRelations?: boolean;
```

Comment:
As we discussed on our little chat earlier today, I think it's slightly cleaner to make this `includeRelations` rather than `exclude`, just to be consistent with `includeFunctions` 🙂

Reply:
Ok, so the default for includeRelations will be true then?

Reply:
I normally tend to have default values be false though .

Reply:
Yeah I know what you're saying, but I think having an include and an exclude simultaneously is slightly messier than having a prop default to true instead of false

Reply:
ok. Done.